### ISSUE TITLE
Val loss: tensor not dropping

### ISSUE THREAD

**Issue by magillos** [GuitarML/Proteus#10]
Apologies if this doesn't belong here as it's not a problem with proteus plug-in itself I have. I've been trying to model my amp but I'm having some troubles. 

What would "Val loss: tensor" not decreasing indicate? The value stays at around 0.75 and model training finishes after 70 epochs or so. I tried to re-run step 4d few times but the results stay similar. 

Also at the beginning of modelling the script shows some errors, not sure how related to the issue they might be:

```
2023-02-19 20:47:46.090475: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib64-nvidia
2023-02-19 20:47:46.090599: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib64-nvidia
2023-02-19 20:47:46.090621: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
unimplemented audio data type conversion...
```

**Comment by GuitarML** [comment]
@magillos No problem, stuck training can happen if the latency (time difference between input/output audio) is too large. And by too large, really more than 5 milliseconds can trip it up. I put a sound click at the beginning of the training audio (and another about 1 second in) to help adjust your recording. Line these clicks up the best you can with your recording in your DAW, export and try again. 

It can also happen if the audio is too difficult for the network to capture, what device are you trying to capture?

**Comment by magillos** [comment]
I aligned the tracks and it worked. The graphs are pretty much identical now. Thanks!
I captured Laney Cub 15R but with low output tube in V1 position installed. I'm happy with the result (I'm using BYOD to load the profile) but I will probably try again with more drive this time. 
I previously tried Tonex to capture this amp but it wasn't even close.
